app/views/users/show.html.erb

```diff
@@ -0,0 +1,43 @@
+<p id="notice"><%= notice %></p>
+
+<p>
+  <strong>Full name:</strong>
+  <%= @user.full_name %>
+</p>
+
+<p>
+  <strong>Username:</strong>
+  <%= @user.username %>
+</p>
+
+<h2> List of <%= @user.username %> creates events: </h2>
+<ul>
+  <% @events.each do |eve| %>
+    <li> Title: <%= eve.title %> Location: <%= eve.location %> </li>
+  <% end %>
```

Comment:
**REQUIRED**
* [ ] The user's show page displays events the user has created and events that the user is attending to separated in past and upcoming **(covered by automated tests)** ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_events_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations)).
Let's separate user created events in past and upcoming.

Reply:
Thank you! Issue solved.